content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -302,6 +302,73 @@ transaction functions. We recommend you get in and out as quick as possible!
 
 </Admonition>
 
+### Transaction timing issues
+
+<Admonition type = "note">
+
+- This section does not apply to MongoDB, because MongoDB does not support [isolation levels](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels).
+- This section does not apply to CockroachDB and SQLite, because these databases only support the highest `Serializable` isolation level.
```

Comment:
```suggestion
- The timing issues discussed in this section do not apply to CockroachDB and SQLite, because these databases only support the highest `Serializable` isolation level.
```